### PR TITLE
W 11339129 improved support for flex version upgrades fa

### PR DESCRIPTION
**Pull request by failup**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/manage-versions.adoc [hunk 1]
```diff
@@ -64,11 +64,33 @@ If an asset in the development state depends on one or more other assets in the
...
 .. To delete a version, select *Delete version*. 
 . Click *Close*.
 
+== Update Policy Implementations
+
+To update policy implementations to be compatible with your Flex Gateway version:
+
+. Go to the *Policies* page.
+. Under *Applied policy status*, you'll see the policies that need updating. Click the more options menu icon and click *Check for implementation updates*.
+. A list of all  available implementations for your current version shows. +
```
Comment (mcordischi) [inline modules/ROOT/pages/manage-versions.adoc]:
```suggestion
. A list of all available implementations for your current version shows. +
```

**Review by tasmoore — DISMISSED** [review]

File: modules/ROOT/pages/manage-versions.adoc [hunk 2]
```diff
@@ -64,11 +64,33 @@ If an asset in the development state depends on one or more other assets in the
...
+. A list of all available implementations for your current version shows. +
+An error message will show if you have no policy implementations compatible with your current Flex Gateway version. See https://docs.mulesoft.com/exchange/add-policy-implementation-files-to-published-policy-definition[Add Policy Implementation Files to Published Policy Definition] for more info.
+. In the confirmation dialog box, click *Update all*.
++ 
+The policy displays under *API-level policies*. 
+
+For automated policies: 
+
+. Go to the *Automated Policies* page.
+. Under *Applied automatic policy status*, you'll see the policies that need updating. Click the more options menu icon and click *See covered APIs* to get a list of the APIs using the incompatible policy.
```
Comment (sankle) [inline modules/ROOT/pages/manage-versions.adoc]:
```suggestion
. Under *Applied automated policy status*, you'll see the policies that need updating. Click the more options menu icon and click *See covered APIs* to get a list of the APIs using the incompatible policy. 
```

File: modules/ROOT/pages/manage-versions.adoc [hunk 3]
```diff
@@ -64,11 +64,33 @@ If an asset in the development state depends on one or more other assets in the
...
 .. To delete a version, select *Delete version*. 
 . Click *Close*.
 
+== Update Policy Implementations
+
+To update policy implementations to be compatible with your Flex Gateway version:
+
+. Go to the *Policies* page.
+. Under *Applied policy status*, you'll see the policies that need updating. Click the more options menu icon and click *Check for implementation updates*.
+. A list of all available implementations for your current version shows. +
```
Comment (fedebrasburg) [inline modules/ROOT/pages/manage-versions.adoc]: Actually, there is no list, users do not choose implementations so what we do is offer the newest compatible version so they can apply it.

File: modules/ROOT/pages/manage-versions.adoc [hunk 4]
```diff
@@ -64,11 +64,33 @@ If an asset in the development state depends on one or more other assets in the
...
+An error message will show if you have no policy implementations compatible with your current Flex Gateway version. See https://docs.mulesoft.com/exchange/add-policy-implementation-files-to-published-policy-definition[Add Policy Implementation Files to Published Policy Definition] for more info.
+. In the confirmation dialog box, click *Update all*.
```
Comment (hernanrivero) [inline modules/ROOT/pages/manage-versions.adoc:81]: We probably need a message here alerting that this action is not rollbackable

**Review by dmerlob — APPROVED** [review]

**Pull request merged**